src/views/Pools/components/CakeVaultCard/index.tsx

```diff
@@ -22,7 +22,11 @@ const StyledCardBody = styled(CardBody)<{ isLoading: boolean }>`
   min-height: ${({ isLoading }) => (isLoading ? '0' : '254px')};
 `
 
-const CakeVaultCard: React.FC<{ pool: Pool; account: string }> = ({ pool, account }) => {
+const CakeVaultCard: React.FC<{ pool: Pool; account: string; showStakedOnly: boolean }> = ({
```

Comment:
Let's turn it into an interface as there are 3 props now?

Reply:
done